### DISCUSSION TITLE
Is this project still maintained?

### DISCUSSION THREAD

**Discussion by MichalisDBA** [dbrest-io/dbREST#7] [General]
Is dbREST still in active development? Will there be any new features?

**Comment by flarco** [comment]
Hello, most of my free time is going into https://github.com/slingdata-io/sling-cli
I wouldn't say it's dead. It's still maintained (it powers [dbnet](https://github.com/dbnet-io/dbnet), which I use daily). But no new features at the moment. 
Feel free to submit PRs.
